src/scripts/main.js

```diff
@@ -354,7 +354,27 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dash = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in your HTML. If it doesn't, this line will cause a runtime error when trying to append rows to a non-existent element.